vllm/worker/hpu_model_runner.py

```diff
@@ -741,6 +749,8 @@ def load_model(self) -> None:
                 get_decoder_layer_suffix(model_config.model_type if
                                          model_config is not None else None),
                 hidden_layer_markstep_interval)
+            recompute_cos_sin = os.getenv('VLLM_COS_SIN_RECOMPUTE',
+                                          'false').lower() == 'true'
```

Comment:
I'd rather have "in ['1', 'true']" instead of "== 'true'"

Reply:
On another note, do you need to get the value of this env var here? Can it be done in __init__ of RotaryEmbedding instead? I don't think it's necessary to pass this value to rope.prepare_cos_sin from here

Reply:
The reason I want to pass the value from hpu_model_runner is because it is easier to be noticed. And I think the "RotaryEmbedding.__init__()" is general for all HW, so I don't want to do any change there.